share/ert/forward-models/res/script/rms

```diff
@@ -1,21 +1,72 @@
 #!/usr/bin/env python
-import sys
+import argparse
 from res.fm.rms import run
 
-run_path = sys.argv[1]
-iens = int(sys.argv[2])
-version = sys.argv[3]
-project = sys.argv[4]
-export_path = sys.argv[5]
-import_path = sys.argv[6]
-workflow = sys.argv[7]
 
-run(iens,
-    project,
-    workflow,
-    run_path,
-    target_file=None,
-    import_path=import_path,
-    export_path=export_path,
-    version=version,
+def _build_argument_parser():
+    description = 'Wrapper script to run rms.'
+    usage = ('The script must be invoked with minimum three positional arguments:\n\n'
```

Comment:
I wonder if the `usage` that `ArgumentParser` automatically calculates matches your `usage` variable (almost) exactly. Maybe drop it?

Reply:
Yes, it matches quite a bit the automatically generated one from ArgumentParser, but that one is only displayed when you are running the script with the `-h` flag. The `usage` is displayed when there is any argarse error like not enough arguments. If you consider it to be a bit overkill I will dorp it.

Reply:
No, keep it—I never knew `usage` had that side-effect.